recipes/bioconductor-ballgown/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: bioconductor-ballgown
+  version: 2.4.2
```

Comment:
No, it should be 2.2.0.

I use the bioconductor skeleton builder to build the skeleton, but if the most recent version of the package isn't available I install it through the R console, get the URL and version info and update the meta.yaml. Obviously I did not do a good job of updating the meta!